package/abs/rancher-abs

```diff
@@ -124,48 +133,73 @@ listening_aliyun_disk() {
 
 # - # https://help.aliyun.com/document_detail/25515.html?spm=5176.doc25513.6.868.xKltry
 attach_aliyun_disk() {
-    log "\t> [disk attach]" /tmp/rancher_abs.log
+    log "\t> [disk attach]"
 
     local delete_with_instance=${1:-"false"}
 
     local aliyun_options="--InstanceId ${INSTANCE_ID} --DiskId ${DISK_ID} --DeleteWithInstance ${delete_with_instance}"
     local aliyun_call_result=$(aliyuncli ecs AttachDisk ${aliyun_options} --output json 2>&1)
     if [ $? -ne 0 ]; then
```

Comment:
Take consideration of the case when the volume is attached to another host here, need to do a `detach` to make the volume available. Regarding issue https://github.com/rancher/rancher/issues/9820